specification/dataprotection/resource-manager/Microsoft.DataProtection/preview/2021-10-01-preview/examples/BackupInstanceOperations/DeleteBackupInstance.json

```diff
@@ -0,0 +1,20 @@
+{
+  "parameters": {
+    "subscriptionId": "04cf684a-d41f-4550-9f70-7708a3a2283b",
+    "resourceGroupName": "000pikumar",
+    "vaultName": "PratikPrivatePreviewVault1",
+    "backupInstanceName": "testInstance1",
+    "api-version": "2021-07-01"
```

Comment:
Could you update the API version in the examples?

Reply:
Updated.